frontends/web/src/utils/bbbaseError.ts

```diff
@@ -0,0 +1,23 @@
+
+/**
+ * Copyright 2019 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { alertUser } from '../components/alert/Alert';
+import { Translate } from '../decorators/translate';
+
+export const bbBaseErrorMessage = (errorCode: string, t: Translate)  => {
```

Comment:
name sounds like it just returns the error message, but it actually does the alert. would be good to add `alert` somewhere in the name.